org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/META-INF/MANIFEST.MF

```diff
@@ -1,25 +1,25 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: Xtext Arithmetics Example - UI
-Bundle-Vendor: Eclipse Xtext
-Bundle-Version: 2.14.0.qualifier
-Bundle-SymbolicName: org.eclipse.xtext.example.arithmetics.ui; singleton:=true
-Bundle-ActivationPolicy: lazy
-Require-Bundle: org.eclipse.xtext.example.arithmetics,
- org.eclipse.xtext.example.arithmetics.ide,
- org.eclipse.xtext.ui,
- org.eclipse.xtext.ui.shared,
- org.eclipse.xtext.ui.codetemplates.ui,
- org.eclipse.ui.editors;bundle-version="3.5.0",
- org.eclipse.ui.ide;bundle-version="3.5.0",
- org.eclipse.ui,
- org.eclipse.compare,
- org.eclipse.xtext.builder,
- org.eclipse.xtend.lib;resolution:=optional,
- org.eclipse.xtext.xbase.lib
-Import-Package: org.apache.log4j
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Export-Package: org.eclipse.xtext.example.arithmetics.ui.internal,
- org.eclipse.xtext.example.arithmetics.ui.quickfix,
- org.eclipse.xtext.example.arithmetics.ui.contentassist
-Bundle-Activator: org.eclipse.xtext.example.arithmetics.ui.internal.ArithmeticsActivator
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Xtext Arithmetics Example - UI
+Bundle-Vendor: Eclipse Xtext
+Bundle-Version: 2.14.0.qualifier
+Bundle-SymbolicName: org.eclipse.xtext.example.arithmetics.ui; singleton:=true
+Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.xtext.example.arithmetics,
+ org.eclipse.xtext.example.arithmetics.ide,
+ org.eclipse.xtext.ui,
+ org.eclipse.xtext.ui.shared,
+ org.eclipse.xtext.ui.codetemplates.ui,
+ org.eclipse.ui.editors;bundle-version="3.5.0",
+ org.eclipse.ui.ide;bundle-version="3.5.0",
+ org.eclipse.ui,
+ org.eclipse.compare,
+ org.eclipse.xtext.builder,
+ org.eclipse.xtend.lib;resolution:=optional,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0"
```

Comment:
Wurde in 2.14 schon API ergänzt, die genutzt wird?

Reply:
nein aber das wird so generiert

Reply:
There is a property in the generator for that. Might have been replaced too greedy.

Reply:
yes but i think it should be in sync anyway

Reply:
ok, might be safer